### PR TITLE
added text and blocks. up next: nesting grids to make text blocks

### PR DESCRIPTION
**Pull request by FizbanUXD**

### CODE REVIEW

File: UXWColors.js [hunk 1]
```diff
@@ -1,13 +1,344 @@
 function UXWColors(props) {
   return (
     <div className="container">
+      <br/>
```
Comment (javag97) [inline UXWColors.js:4]: **praise**: section looks good to me

**suggestion**: if you're trying to space things out, you can add [some padding](https://tailwindcss.com/docs/padding) to any HTML element

File: UXWColors.js [hunk 2]
```diff
@@ -1,13 +1,344 @@
...
+      and eccentric colors. This color palette has been carefully chosen
+      to provide versatility and function while still providing boundless
+      room for expression.
+      </p>
+
+      <br/>
+      <br/>
+      <br/>
+
+      <div className="font-bold font-2xl">
```
Comment (javag97) [inline UXWColors.js:29]: **suggestion**: you can change this to something like 

```html
<h2 className="font-bold font-2xl">Using our palette</h2> 
```

You can add classes to **any** HTML element

File: UXWColors.js (hunk 1 above)
Comment (javag97) [reply UXWColors.js:4]: What I would do is substitute a `pb-4` to the previous HTML element for every `<br/>`. For example, if you you're using 3 `<br/>` tags in succession,  I would add a `pb-12` to the HTML tag _before_ the `<br/>` tags.

File: UXWColors.js [hunk 3]
```diff
@@ -1,13 +1,344 @@
...
+               </div>
+             </div>
+
+           </div>
+
+           <br/>
+           <br/>
+           <br/>
+
+
```
Comment (javag97) [inline UXWColors.js:96]: Are you using VSCode? From the editor: If you right click within the contents of the file itself, you can press `Format Document` to have your text editor resolve formatting

[Link to Stack Overflow Thread](https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode)

**Review by javag97 — CHANGES_REQUESTED** [review]

File: UXWColors.js (hunk 2 above)
Comment (FizbanUXD) [reply UXWColors.js:29]: oh, maybe a silly question, but what is a div then? I thought that was like... the tag that means tell it to change something but I guess that's wrong?

**Review by javag97 — APPROVED** [review]

**Pull request merged**